app/views/layouts/nested_node_resource.html.erb

```diff
@@ -1,5 +1,7 @@
 <% content_for :content do %>
-  <%= render "nodes/sidebar" %>
+  <% content_for :sidebar do %>
```

Comment:
does `content_for` need to be nested inside the other `content_for`? Seems a bit untidy